saw-central/src/SAWCentral/Builtins.hs

```diff
@@ -2376,14 +2397,14 @@ setMonadification sc cry_str saw_str poly_p =
        liftIO $
        case Map.lookup cry_nm (CEnv.eExtraTypes $ rwCryptol rw) of
          Just schema ->
-           -- putStrLn ("Found Cryptol type for name: " ++ show cry_str) >>
+           -- putStrLn $ Text.unpack $ "Found Cryptol type for name: " <> show cry_str >>
```

Comment:
We could use [`Data.Text.IO.putStrLn`](https://hackage-content.haskell.org/package/text-2.1.2/docs/Data-Text-IO.html#v:putStrLn) here instead of `Prelude.putStrLn . Data.Text.unpack` (and similarly below).